matter_server/server/device_controller.py

```diff
@@ -176,17 +184,22 @@ async def commission_with_code(self, code: str) -> MatterNodeData:
         retries = 3
         while retries:
             try:
+                LOGGER.info(
```

Comment:
why? this is also logged in the interview method

Reply:
I see. But in debug level only. I think this doesn't happen all to often, so loggin in info seems sensible to me. Should we raise in `interview_node`?

Reply:
yes please, that makes more sense